### ISSUE TITLE
Mobile drawing experience

### ISSUE THREAD

**Issue by Jacob-Griffin** [Jacob-Griffin/TelephonePictionary2.0#25]
We need to have some way where mobile users aren't drawing a landscape picture on a portrait screen. This is going to take some doing with making sure they can pop out the canvas, and forcing their browser to do landscape for only that game state. This will also depend on #7, since having it popped out means they shouldn't have to scroll

**Comment by Jacob-Griffin** [comment]
Landscape potential for mobile drawing is becoming essential, given small screens, and submit button needs to be accessible without scroll. See task 1 in issue #27

**Comment by Jacob-Griffin** [comment]
Finally got around to this one. I've now placed canvas and canvas tools side-by side in general to better use screenspace, then double checked that that works on mobile. To guide users toward this experience, I made an extra little notification to turn your phone to landscape mode. We should test if the thresholds are reasonable for thin desktop windows and if the layout feels right, but I've got an iteration out there
